doc/source/using-ray-on-a-large-cluster.rst

```diff
@@ -1,309 +0,0 @@
-Manual Cluster Setup on a Large Cluster
```

Comment:
This should be subsumed by autoscaler

Reply:
If it is, that's awesome, but is it actually?

When using parallel-ssh, you are starting the cluster from the head node. When using the autoscaler, *I think* you are starting the cluster from your laptop which has to ssh to the head node. These are almost the same thing but not quite. I wonder if we're eliminating some use cases.

Are there other differences? Or is that the only one?

Reply:
> If it is, that's awesome, but is it actually?
I'm pretty sure they're effectively the same.

> When using parallel-ssh, you are starting the cluster from the head node.
The `worker_setup_commands/worker_ray_commands` are run from the head node (not from your laptop)

> with the autoscaler, you are starting the cluster from your laptop which has to ssh to the head node.
The head node is started from your laptop. However, the rest of the nodes are not and do not require you (on your laptop) to hold a bash session on the head node.